src/app/admin-icon-input/admin-icon-input.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-form-field class="full-width" floatLabel="always">
+  <mat-label>
+    <span i18n>Icon</span>
+    &nbsp;
+    <fa-icon
+      icon="question-circle"
+      matTooltip="The icon to represent this entity type, e.g. when displaying records as a small preview block."
+      i18n-matTooltip
```

Comment:
If we want to use this component more widely, this tooltip is too specific. Removing it here.
```suggestion
```